sky-bench/src/cli.rs

```diff
@@ -0,0 +1,79 @@
+use clap::ArgAction;
+use clap::Parser;
```

Comment:
Can we merge the imports into one-line to match the import styles? Like this:
```rust
use clap::{ArgAction, Parser};
```
Thanks!